test/Conversion/VectorToXeGPU/gemm_3x3.mlir

```diff
@@ -0,0 +1,114 @@
+// RUN: %python_executable %imex_runner --requires=l0-runtime -i %s --pass-pipeline-file=%p/vector-to-llvm.pp \
```

Comment:
This test in an Integration test running on GPU.
I would suggest placing in somewhere like under
test/Integration/Dialect/Vector/